saleor/search/backends/elasticsearch.py

```diff
@@ -463,12 +482,14 @@ def _do_search(self):
                 setattr(obj, self._score_field, scores.get(str(obj.pk)))
 
         # Return results in order given by Elasticsearch
-        return [results[str(pk)] for pk in pks if results[str(pk)]]
+        return ([results[str(pk)] for pk in pks if results[str(pk)]],
```

Comment:
Complexity of return value execution is far to high for human recognition. I would suggest decomposing it to named pre-evaluated variables simply returned at return statement.